### PR TITLE
Add description of fringes for the browse images in InSAR product guide

### PR DESCRIPTION
**Pull request by hjkristenson**

### CODE REVIEW

File: docs/guides/insar_product_guide.md [hunk 1]
```diff
@@ -257,7 +257,10 @@ All of the main InSAR product files are 32-bit floating-point single-band GeoTIF
 
 If the **water mask** option is selected, the water mask is applied prior to phase unwrapping to exclude water pixels from the process. The water mask is not precise; the coastal shorelines from the [GSHHG](http://www.soest.hawaii.edu/wessel/gshhg/land) are buffered out to 3 km from shore to reduce the possibility of near-shore features being excluded while reducing the number of water pixels that may impact the quality of the phase unwrapping process. Inland waters are not masked.
 
-**Browse images** are included for the wrapped (color_phase) and unwrapped (unw_phase) phase files, which are in PNG format and are each 2048 pixels wide. 
+**Browse images** are included for the wrapped (color_phase) and unwrapped (unw_phase) phase files, which are in PNG format and are each 2048 pixels wide. The browse images are displayed using a cyclic color ramp to generate fringes. 
+
+- Each fringe in a wrapped (color_phase) browse image represents a 2-pi phase difference, and the line-of-sight displacement for each fringe is equivalent to half the wavelength of the sensor.
```
Comment (asjohnston-asf) [inline docs/guides/insar_product_guide.md:262]: is it worth converting 2-pi to centimetres here, like in the `unw_phase` description?

**Review by asjohnston-asf — APPROVED** [review]

File: docs/guides/insar_product_guide.md (hunk 1 above)
Comment (hjkristenson) [reply docs/guides/insar_product_guide.md]: @asjohnston-asf Check out the change in [this commit](https://github.com/ASFHyP3/hyp3-docs/pull/174/commits/6ffef825890a0fab1f053c95c161e57086f8ab5c). 
I thought maybe it was too wordy, but the advantage is that the approximate sensor wavelength is documented for reference. I took a stab at combining the statements to shorten things up, but didn't come up with anything I liked better.
